ale/drivers/selene_drivers.py

```diff
@@ -216,27 +216,6 @@ def detector_center_sample(self):
         """
         return spice.gdpool('INS{}_CENTER'.format(self.ikid), 0, 2)[0] - 0.5
 
-
-    @property
-    def reference_frame(self):
-        """
-        Kaguya uses a slightly more accurate "mean Earth" reference frame for
-        moon obvervations. see https://darts.isas.jaxa.jp/pub/spice/SELENE/kernels/fk/moon_assoc_me.tf
-
-        Expects target_name to be defined. This should be a string containing the
-        name of the target body.
-
-        Returns
-        -------
-        : str
-          Reference frame
-        """
-        if self.target_name.lower() == "moon":
-            return "MOON_ME"
-        else:
-            # TODO: How do we handle no target?
-            return "NO TARGET"
-
```

Comment:
Changes from #302 mean this is no longer needed. It will now pull whatever your kernels say should be used for the Moon